docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -471,6 +471,31 @@
           "default": false
         }
       }
+    },
+    "extensions": {
+      "type": "object",
+      "description": "Extends various parts of the Grafana UI with commands or links.",
+      "additionalProperties": false,
+      "properties": {
+        "links": {
+          "type": "array",
+          "description": "Expose a page link that can be used by Grafana core or other plugins to navigate users to the plugin",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "description": {
+                "type": "string"
+              },
+              "path": {
+                "type": "string"
+              }
```

Comment:
Question: in the [design doc](https://docs.google.com/document/d/1-g6-4RJx3LzsldvdwUZb_wHXAS5Z7_kt6Ql9zAoLj8I/edit#) we wanted to have a `schema` property as well. Would it make sense to add it here as an optional property now or you were planning to add it later?

Reply:
We are planning on adding it later when we have a better understanding of how the schema solution can be added in this context.